src/pages/Profile/Profile.js

```diff
@@ -1,6 +1,5 @@
 import React, { Component } from 'react';
 import Formsy, { HOC } from 'formsy-react';
-import { Header } from '../../components/Auth/Header/Header';
 import Field from '../../components/Auth/Field/Field';
```

Comment:
Why did you import the component, but did not use it in the code? But copied from that component a bunch of extra code ...